rdkit/Chem/MolDb/Loader_orig.py

```diff
@@ -153,12 +153,12 @@ def LoadDb(suppl, dbName, nameProp='_Name', nameCol='compound_id', silent=False,
   curs = conn.GetCursor()
   if startAnew:
     try:
-      curs.execute(f'drop table {regName}' % regName)
+      curs.execute(f'drop table {regName}')
     except Exception:
       pass
-    curs.execute(f'create table {regName} {",".join(typs)}')
+    curs.execute(f'create table {regName} ({",".join(typs)})')
```

Comment:
Not identical:
```
>>> f'create table {regName} {",".join(typs)}'
'create table abc d,e,f'
>>> f'create table {regName} ({",".join(typs)})'
'create table abc (d,e,f)'
```

Reply:
I don't understand, the original code in line 160 has round bracket. This type of error raised once in the first commit after initial review. You can see below